src/Sylius/Bundle/ProductBundle/Doctrine/ORM/ProductAttributeValueRepository.php

```diff
@@ -27,8 +29,18 @@ class ProductAttributeValueRepository extends EntityRepository implements Produc
 {
     public function findByJsonChoiceKey(string $choiceKey): array
     {
-        return $this->createQueryBuilder('o')
-            ->andWhere('o.json LIKE :key')
+        $connection = $this->getEntityManager()->getConnection();
+        $isPostgres = $this->isPostgres($connection);
+
+        $qb = $this->createQueryBuilder('o');
```

Comment:
Full name matches existing codebase better.

```suggestion
        $queryBuilder = $this->createQueryBuilder('o');
```